content/docs/release-notes.md

```diff
@@ -2,7 +2,54 @@
 title: Release Notes
 ---
 
-## 2022-07-11
+## 2022-07-19 Storage release
+
+### What's new
+
+* Safekeeper: Back up WAL to S3 for disaster recovery.
+
+* Safekeeper: Download WAL from S3 on demand.
+
+* Safekeeper: Replace callmemaybe with etcd subscriptions on safekeeper timeline info.
```

Comment:
```suggestion
* Safekeeper: replace callmemaybe with etcd subscriptions on safekeeper timeline info.
```

Reply:
I suggest to put `callmemaybe` in accents and explicitly call out what it is. call? internal api call? internal api call from safekeeper to pageserver?